Jenkinsfile.groovy

```diff
@@ -9,14 +9,14 @@ if (currentBuild.rawBuild.getCauses().toString().contains('BranchIndexingCause')
   return
 }
 
-library identifier: 'jsadmin_pipeline@next', retriever: modernSCM([
- $class: 'GitSCMSource',
- remote: 'git@bitbucket.org:coveord/jsadmin_pipeline.git',
- credentialsId: 'coveo-bitbucket-rd-ssh'
-])
+library(
+    identifier: "jsadmin_pipeline@master",
+    retriever: modernSCM(github(credentialsId: "github-app-dev", repository: "jsadmin_pipeline", repoOwner: "coveo")),
```

Comment:
Not sure but we have `github-commit-token` in admin-ui right?

Reply:
This is only to get access to the jsadmin_pipeline repos and the credential github-commit-token is to be used only when necessary